src/SeqViz/Circular/Annotations.tsx

```diff
@@ -14,9 +41,9 @@ import { COLOR_BORDER_MAP, darkerColor } from "../../utils/colors";
  *
  * @type {Function}
```

Comment:
you've done a good job of this already, but would remove whatever shoddy jsdocs you see lying around. Was from a time of Flow (and even slightly before then)